src/AsyncCreatable.js

```diff
@@ -7,19 +7,42 @@ const AsyncCreatable = React.createClass({
 	render () {
 		return (
 			<Select.Async {...this.props}>
-				{(asyncProps) => (
-					<Select.Creatable {...this.props}>
-						{(creatableProps) => (
-							<Select
-								{...asyncProps}
-								{...creatableProps}
-							/>
-						)}
+				{({ref, ...asyncProps}) => {
+					const asyncRef = ref
+					return <Select.Creatable {...asyncProps} >
+						{({ref, ...creatableProps}) => {
+							const creatableRef = ref
+							return this.props.children({
+								...creatableProps,
+								ref: (select) => {
+									creatableRef(select)
+									asyncRef(select)
+									this.select = select
+								}
+							})
+						}}
```

Comment:
https://github.com/realestate-com-au/react-select/blob/535d1b6ef1a7d62ad50dd84f0165c77bcf10d293/src/Async.js#L173 keeps a ref to the `Select`, as does https://github.com/realestate-com-au/react-select/blob/535d1b6ef1a7d62ad50dd84f0165c77bcf10d293/src/Creatable.js#L209